src/test/java/cucumber/steps/StepDefinitions.java

```diff
@@ -0,0 +1,46 @@
+package cucumber.steps;
+
+import controllers.DashboardController;
+import controllers.LaunchController;
+import io.cucumber.java.en.When;
+
+import java.util.ArrayList;
+
+import static helpers.UserTokenGenerator.getToken;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItem;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class StepDefinitions {
+    private LaunchController launch = new LaunchController(Hooks.api);
+    private DashboardController dashboards = new DashboardController(Hooks.api);
+    private ArrayList<String> listWithDashboardNames;
+
+    @When("use is logged in")
+    public void useIsLoggedIn() {
+        assertNotNull(getToken(), "Authorization failed");
+    }
+
+    @When("user makes request to get all permitted dashboard resources for the project")
+    public void getAllPermittedDashboardResourcesForTheProject() {
+        listWithDashboardNames = dashboards.getAllDashboardNames();
+    }
+
+    @When("user sees dashboard with {string} name")
+    public void getDashboardNameFromList(String dashboardName) {
```

Comment:
I'm not sure but maybe better methods with asserts will names like 'checkSomethingImportant'?